README.md

```diff
@@ -15,31 +15,193 @@ Before you begin, you need to install the following tools:
+
+🔏 You can also check out your smart contract `YourCollectible.sol` in `packages/hardhat/contracts`.
+
+💼 Take a quick look at your deploy script `00_deploy_your_contract.js` in `packages/hardhat/deploy`.
+
+📝 If you want to make frontend edits, Open and look for the page you want to edit in `packages/nextjs/pages` example `myNFTs.tsx`.
+
+---
+
+## Checkpoint 3: 💾 Deploy it! 🛰
+
+🛰 Ready to deploy to a public testnet?!?
+
+> Change the defaultNetwork in `packages/hardhat/hardhat.config.ts` to `sepolia`
+
+![image](https://github.com/scaffold-eth/se-2-challenges/assets/80153681/9ac9e2be-86cb-4421-96f3-6065925987ed)
+
+🔐 Generate a deployer address with `yarn generate`
+![image](https://github.com/scaffold-eth/se-2-challenges/assets/80153681/e836ce34-c8c2-4029-831a-dfc82fe6f26b)
+
+👛 View your deployer address using the `yarn account`
+![image](https://github.com/scaffold-eth/se-2-challenges/assets/80153681/e743816b-f587-424d-a21d-96088cb3925b)
+
+⛽️ Use a faucet like [allthatnode.com/faucet/ethereum.dsrv](https://allthatnode.com/faucet/ethereum.dsrv) or [web.getlaika.app/faucets](https://web.getlaika.app/faucets) to fund your deployer address.
+
+> ⚔️ Side Quest: Keep a 🧑‍🎤 [punkwallet.io](https://punkwallet.io) on your phone's home screen and keep it loaded with testnet eth. 🧙‍♂️ You'll look like a wizard when you can fund your deployer address from your phone in seconds.
+
+🚀 Deploy your NFT smart contract:
+
+```shell
+yarn deploy
+```
+
+> 💬 Hint: You can set the `defaultNetwork` in `hardhat.config.ts` to `sepolia` **OR** you can `yarn deploy --network sepolia`.
+
+---
+
+## Checkpoint 4: 🚢 Ship it! 🚁
+
+> ✏️ Edit your frontend config `scaffold.config.ts` in `packages/nextjs/scaffold.config.ts` to change the `targetNetwork` to `chains.seplolia` :
+
+![image](https://github.com/scaffold-eth/se-2-challenges/assets/80153681/34de5be1-8cf1-4564-ba4b-358824d94547)
+
+> You should see the correct network in the frontend (http://localhost:3000):
+
+![image](https://github.com/scaffold-eth/se-2-challenges/assets/80153681/50eef1f7-e1a3-4b3b-87e2-59c19362c4ff)
+
+> 🦊 You will need to connect your wallet, by default 🔥 `buner wallet` are only available on `hardhat` network but you can enable them on every chain by setting `onlyLocal : false` inside `burnerWallet` in your frontend config `scaffold.config.ts` in `packages/nextjs/scaffold.config.ts`
+
+![image](https://github.com/scaffold-eth/se-2-challenges/assets/80153681/f582d311-9b57-4503-8143-bac60346ea33)
+
+> NOTE: Before deploying you should always make sure you update `fromBlock` config for `useScaffoldEventHistory` hook to the `blocknumber - 10` at which your contract was deployed. Lets update the `fromBlock` of `useScaffoldContractEventHistory` in [`packages/nextjs/pages/trasfer.tsx`](https://github.com/scaffold-eth/se-2-challenges/blob/c18ed9bd202b8614da6775a189937b4facb58929/packages/nextjs/pages/transfers.tsx#L11) eg.`fromBlock:3750241`
```

Comment:
Don't know if this feels too much 😅, Mabe we should make it shorter? (maybe just keep the second point only? )

Reply:
I've been thinking about this and not sure what's best.

I think is not super important here, since these aren't going to be production-level applications, so the transfers page being slow (when deployed to a testnet) shouldn't be a big deal. I think in SE-1 it's also fetching from block 0, right? It might be too specific for a beginner. 

Maybe we could add an in-code comment on the `transfer.tsx` page?

Reply:
Oh, now that I tested it..... it's really slow :sweat_smile: 

Maybe we could rephrase it a bit and make it look like a suggestion / optimization....instead of "you should always"... something more... "if you want you transfer page to load faster, you can...."

Reply:
Update at [7cc9177](https://github.com/scaffold-eth/se-2-challenges/pull/5/commits/7cc91779b1aa3420c32bab83ec2900fc065bf365) put please feel to re-update it,  Pablo or Carlos 🙌

Reply:
The new version of the HINT is much better, nice rephrase 👌